[
  {
    "log": "2014-06-09 15:56:40,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354600073
  },
  {
    "log": "2014-06-09 15:56:40,075 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354600075
  },
  {
    "log": "2014-06-09 15:56:45,075 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354605075
  },
  {
    "log": "2014-06-09 15:56:45,076 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354605076
  },
  {
    "log": "2014-06-09 15:56:50,075 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354610075
  },
  {
    "log": "2014-06-09 15:56:50,078 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354610078
  },
  {
    "log": "2014-06-09 15:56:55,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354615073
  },
  {
    "log": "2014-06-09 15:56:55,074 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354615074
  },
  {
    "log": "2014-06-09 15:57:00,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354620073
  },
  {
    "log": "2014-06-09 15:57:00,074 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354620074
  },
  {
    "log": "2014-06-09 15:57:05,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354625073
  },
  {
    "log": "2014-06-09 15:57:05,075 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354625075
  },
  {
    "log": "2014-06-09 15:57:10,075 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354630075
  },
  {
    "log": "2014-06-09 15:57:10,077 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354630077
  },
  {
    "log": "2014-06-09 15:57:15,077 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354635077
  },
  {
    "log": "2014-06-09 15:57:15,079 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354635079
  },
  {
    "log": "2014-06-09 15:57:20,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354640073
  },
  {
    "log": "2014-06-09 15:57:20,075 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354640075
  },
  {
    "log": "2014-06-09 15:57:25,074 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354645074
  },
  {
    "log": "2014-06-09 15:57:25,075 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354645075
  },
  {
    "log": "2014-06-09 15:57:30,074 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354650074
  },
  {
    "log": "2014-06-09 15:57:30,076 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354650076
  },
  {
    "log": "2014-06-09 15:57:35,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354655073
  },
  {
    "log": "2014-06-09 15:57:35,075 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354655075
  },
  {
    "log": "2014-06-09 15:57:40,077 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354660077
  },
  {
    "log": "2014-06-09 15:57:40,079 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354660079
  },
  {
    "log": "2014-06-09 15:57:45,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354665073
  },
  {
    "log": "2014-06-09 15:57:45,076 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354665076
  },
  {
    "log": "2014-06-09 15:57:50,075 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354670075
  },
  {
    "log": "2014-06-09 15:57:50,076 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354670076
  },
  {
    "log": "2014-06-09 15:57:55,074 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354675074
  },
  {
    "log": "2014-06-09 15:57:55,077 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354675077
  },
  {
    "log": "2014-06-09 15:58:00,075 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354680075
  },
  {
    "log": "2014-06-09 15:58:00,077 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354680077
  },
  {
    "log": "2014-06-09 15:58:05,072 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354685072
  },
  {
    "log": "2014-06-09 15:58:05,073 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354685073
  },
  {
    "log": "2014-06-09 15:58:10,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354690073
  },
  {
    "log": "2014-06-09 15:58:10,074 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354690074
  },
  {
    "log": "2014-06-09 15:58:15,074 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354695074
  },
  {
    "log": "2014-06-09 15:58:15,075 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354695075
  },
  {
    "log": "2014-06-09 15:58:20,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354700073
  },
  {
    "log": "2014-06-09 15:58:20,074 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354700074
  },
  {
    "log": "2014-06-09 15:58:25,072 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354705072
  },
  {
    "log": "2014-06-09 15:58:25,073 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354705073
  },
  {
    "log": "2014-06-09 15:58:30,078 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354710078
  },
  {
    "log": "2014-06-09 15:58:30,080 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354710080
  },
  {
    "log": "2014-06-09 15:58:35,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354715073
  },
  {
    "log": "2014-06-09 15:58:35,074 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354715074
  },
  {
    "log": "2014-06-09 15:58:40,073 - ERROR [single-log-reader-2:c.c.l.r.AvroFileLogReader@165] - Got exception while reading log file file:/Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) [watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] ",
    "offset": 1402354720073
  },
  {
    "log": "2014-06-09 15:58:40,074 - ERROR [single-log-reader-2:c.c.l.r.StandaloneLogReader$2@216] - Got exception: java.lang.RuntimeException: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at com.google.common.base.Throwables.propagate(Throwables.java:160) ~[guava-13.0.1.jar:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:166) ~[watchdog/:na] at co.cask.cdap.logging.read.StandaloneLogReader$2.run(StandaloneLogReader.java:203) ~[watchdog/:na] at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439) [na:1.6.0_65] at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303) [na:1.6.0_65] at java.util.concurrent.FutureTask.run(FutureTask.java:138) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895) [na:1.6.0_65] at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918) [na:1.6.0_65] at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65] Caused by: java.io.FileNotFoundException: /Users/rohit/Documents/system/data/logs/avro/developer/AutoPerkApp/flow-AutoPerkFlow/2013-11-26/1385510953427.avro (No such file or directory) at java.io.FileInputStream.open(Native Method) ~[na:1.6.0_65] at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:120) ~[na:1.6.0_65] at org.apache.twill.filesystem.LocalLocation.getInputStream(LocalLocation.java:76) ~[twill-common-0.3.0-incubating-SNAPSHOT.jar:0.3.0-incubating-SNAPSHOT] at co.cask.cdap.logging.read.AvroFileLogReader$LocationSeekableInput.&lt;init&gt;(AvroFileLogReader.java:185) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.createReader(AvroFileLogReader.java:171) ~[watchdog/:na] at co.cask.cdap.logging.read.AvroFileLogReader.readLogPrev(AvroFileLogReader.java:95) ~[watchdog/:na] ... 7 common frames omitted ",
    "offset": 1402354720074
  }
]